src/plugins/CoreLoadPlugin.ts

```diff
@@ -58,7 +58,7 @@ function isContextual(module: NormalModule, issuers: string[]): boolean {
  * The updated path.
  */
 function stripPath(basePath: string, path: string): string {
-	let resolved = path.replace(basePath + '/', '').replace(/\..*$/, '');
+	let resolved = (basePath ? path.replace(basePath + '/', '') : path).replace(/\..*$/, '');
```

Comment:
When testing the generated ES6 code, the code coverage generation is actually dropping the default value `''` for basePath in the object destructuring. That was hiding a bug where if `basePath` was `''` we would remove the leading slash from a path. It was hidden because instead we were calling replace with `'undefined/'`. 

I discovered this when I switched to targeting ES5, although it's back to ES6 now so we would only see this if running tests without coverage.